spellbook/models/nft/nft_schema.yml

```diff
@@ -1,34 +1,286 @@
 version: 2
 
 models:
-  - name: nft_trades
+  - name: nft_transactions
     meta:
       blockchain: ethereum, solana
       sector: nft
       contributors: soispoke
     config:
-      tags: ['nft', 'opensea', 'magiceden', 'ethereum', 'solana', 'trades']
+      tags: ['nft', 'opensea', 'magiceden', 'ethereum', 'solana', 'transactions']
     description: >
-        NFT trades
+        NFT transactions
     columns:
       - &blockchain
         name: blockchain
         description: "Blockchain"
-      - name: project
+      - &project
+        name: project
         description: "Project"
-      - name: version
+      - &version
+        name: version
         description: "Project version"
-      - name: tx_hash
-        description: "Transaction hash"
-      - name: block_time
+      - &block_time
+        name: block_time
         description: "UTC event block time"
-      - name: token_symbol
-        description: "Token Symbol"
-      - name: token_address
-        description: "Token Address"
-      - name: amount
-        description: "Value of the trade at time of execution in the original currency"
-      - name: amount_usd
+      - &token_id
+        name: token_id
+        description: "NFT Token ID"
+      - &collection
+        name: collection
+        description: "NFT collection name"
+      - &amount_usd
+        name: amount_usd
         description:  "USD value of the trade at time of execution"
-      - name: unique_trade_id
-        description:  "Unique trade ID"
\ No newline at end of file
+      - &token_standard
+        name: token_standard
+        description:  "Token standard"
+      - &trade_type
+        name: trade_type
+        description: "Identify whether it was a single NFT trade or multiple NFTs traded"
+      - &number_of_items
+        name: number_of_items
+        description:  "Number of items traded"
+      - &trade_category
+        name: trade_category
+        description:  "How was this NFT traded ? (Direct buy, auction, etc...)"
+      - &evt_type
+        name: evt_type
+        description:  "Event type (Trade, Mint, Burn)"
+      - &seller
+        name: seller
+        description:  "Seller wallet address"
+      - &buyer
+        name: buyer
+        description:  "Buyer wallet address"
+      - &amount_original
+        name: amount_original
+        description:  "Traded amount in original currency"
+      - &amount_raw
+        name: amount_raw
+        description:   "Traded amount in original currency before decimals correction"
+      - &currency_symbol
+        name: currency_symbol
+        description:  "Symbol of original currency used for payment"
+      - &currency_contract
+        name: currency_contract
+        description:  "Contract address of original token used for payment, with ETH contract address swapped for WETH"
+      - &currency_contract_original
```

Comment:
If `currency_contract` is the same except for the ETH -> WETH mapping, is `currency_contract_original` really needed? You can always easily distinguish ETH from WETH through `currency_symbol` 🤔

Reply:
Definitely a good point thanks! I'll remove the currency_contract_original column as suggested 👍